examples/example-next-static-export/package.json

```diff
@@ -20,11 +21,13 @@
     "react-dom": "^18"
   },
   "devDependencies": {
+    "@next/env": "^14.2.7",
     "@types/node": "^20",
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "eslint": "^8.57.0",
     "eslint-config-next": "^14.2.3",
-    "typescript": "^5.5.4"
+    "typescript": "^5.5.4",
+    "vitest": "^2.0.5"
```

Comment:
Yes, there is a build process now that combines the dependencies into built template files:
https://github.com/palantir/osdk-ts/blob/main/packages/create-app.template.react/README.md

I believe this was to make it easier to make dependabot work on a real package.json file rather than the handlebars file